docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,639 @@
+openapi: 3.0.3
+info:
+  title: Firma con IO - Issuer API
+  version: 1.0.0
+servers:
+  - url: https://api.io.pagopa.it/api/v1/sign
+    description: production
+security:
+  - SubscriptionKey: []
+paths:
```

Comment:
not urgent but you probably want to create "Links" sections with the returned URLs to store (upload) documents: https://swagger.io/docs/specification/links/